Recipes/Apply_function_to_every_gridpoint.ipynb

```diff
@@ -37,12 +37,13 @@
   },
```

Comment:
<p>I know its not really in scope, but this would look better on <a href="https://cosima-recipes.readthedocs.io/en/latest/recipes.html" target="_blank">https://cosima-recipes.readthedocs.io/en/latest/recipes.html</a> if the title matched the filename (e.g. the title is just _Apply_function_to_every_gridpoint_)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done